lib/event-handler.js

```diff
@@ -3,6 +3,7 @@
 const v02 = require('cloudevents-sdk/v02');
 const v03 = require('cloudevents-sdk/v03');
 const v1 = require('cloudevents-sdk/v1');
+const Spec = require('./ce-constants.js').Spec;
```

Comment:
Nit: This would be more concise as

 ```suggestion
const { Spec } = require('./ce-constants.js');
```
But that's really minor.